### DISCUSSION TITLE
What does bundle mean?[General] <Title>

### DISCUSSION THREAD

**Discussion by workforfun2000** [zkSync-Community-Hub/zksync-developers#730] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

![2024-10-14 17 52 14](https://github.com/user-attachments/assets/7345091a-1dc7-466b-b3a3-8acf8097d619)
This picture is the page for depositing zk tokens on OKEx. What do bundle 1 and bundle 2 in the picture mean?

**Comment by alhenaai** [comment]
In the context of depositing ZK tokens on OKEx, "bundle 1" and "bundle 2" likely refer to specific groups or sets of transactions that are processed together. Bundling transactions can help in optimizing the processing times and fees associated with the transfers on the blockchain.

For more specific details about what each bundle includes or represents, it would be best to consult OKEx's support or documentation directly, as the specifics can vary based on their internal processes and the design of their deposit system.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by workforfun2000** [comment]
For an exchange, how many blocks or what conditions are required for a deposit to be considered safe and credited to the user's account?

**Reply by kaymomin** [reply, marked as answer]
The confirmation count for ETH & ERC20 token deposits typically falls between 32 and 64 blocks (as per their last community announcement) before being credited on OKX. 

For more precise details, please refer to OKEx’s support or their official documentation.

Closing this ticket for now, but feel free to reach out if you need anything else.

**Reply by workforfun2000** [reply]
Does 32 or 64 blocks refer to the Ethereum network?

**Reply by kaymomin** [reply]
Yes, this is specific to ETH network.
